appinfo/app.php

```diff
@@ -60,12 +60,7 @@
 $groupBackend = new \OCA\User_SAML\GroupBackend(\OC::$server->getDatabaseConnection());
 \OC::$server->get(IGroupManager::class)->addBackend($groupBackend);
 
-$samlSettings = new \OCA\User_SAML\SAMLSettings(
-	$urlGenerator,
-	$config,
-	$request,
-	$session
-);
+$samlSettings = \OC::$server->query(\OCA\User_SAML\SAMLSettings::class);
```

Comment:
```suggestion
$samlSettings = \OC::$server->get(\OCA\User_SAML\SAMLSettings::class);
```

Reply:
Thanks a lot, you are right, query is deprecated